### PR TITLE
Add export inference_graph

### PR DESCRIPTION
**Pull request by sguada**

### CODE REVIEW

File: slim/README.md [hunk 1]
```diff
@@ -204,7 +206,6 @@ Model | TF-Slim File | Checkpoint | Top-1 Accuracy| Top-5 Accuracy |
 [MobileNet_v1_1.0_224](https://arxiv.org/pdf/1704.04861.pdf)|[Code](https://github.com/tensorflow/models/blob/master/slim/nets/mobilenet_v1.py)|[mobilenet_v1_1.0_224_2017_06_14.tar.gz](http://download.tensorflow.org/models/mobilenet_v1_1.0_224_2017_06_14.tar.gz)|70.7|89.5|
 [MobileNet_v1_0.50_160](https://arxiv.org/pdf/1704.04861.pdf)|[Code](https://github.com/tensorflow/models/blob/master/slim/nets/mobilenet_v1.py)|[mobilenet_v1_0.50_160_2017_06_14.tar.gz](http://download.tensorflow.org/models/mobilenet_v1_0.50_160_2017_06_14.tar.gz)|59.9|82.5|
 [MobileNet_v1_0.25_128](https://arxiv.org/pdf/1704.04861.pdf)|[Code](https://github.com/tensorflow/models/blob/master/slim/nets/mobilenet_v1.py)|[mobilenet_v1_0.25_128_2017_06_14.tar.gz](http://download.tensorflow.org/models/mobilenet_v1_0.25_128_2017_06_14.tar.gz)|41.3|66.2|
-
```
Comment (korrawat) [inline slim/README.md]: This makes the [table](https://github.com/tensorflow/models/blob/master/slim/README.md#pre-trained-models) display incorrectly by including the next line in the first column of the next row. I think there should be an empty line after the table.

File: slim/README.md [hunk 2]
```diff
@@ -327,8 +328,72 @@ $ python eval_image_classifier.py \
...
+```shell
+$ python export_inference_graph.py \
+  --alsologtostderr \
+  --model_name=inception_v3 \
+  --output_file=/tmp/inception_v3_inf_graph.pb
+
+$ python export_inference_graph.py \
+  --alsologtostderr \
+  --model_name=mobilenet_v1 \
+  --image_size=224 \
```
Comment (futurely) [inline slim/README.md]: `slim/export_inference_graph.py` uses `default_image_size`.

File: slim/export_inference_graph.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  if not FLAGS.output_file:
+    raise ValueError('You must supply the path to save to with --output_file')
+  tf.logging.set_verbosity(tf.logging.INFO)
+  with tf.Graph().as_default() as graph:
+    dataset = dataset_factory.get_dataset(FLAGS.dataset_name, 'validation',
+                                          FLAGS.dataset_dir)
+    network_fn = nets_factory.get_network_fn(
+        FLAGS.model_name,
+        num_classes=(dataset.num_classes - FLAGS.labels_offset),
+        is_training=FLAGS.is_training)
```
Comment (futurely) [inline slim/export_inference_graph.py]: MobileNet needs another parameter [`depth_multiplier`](https://github.com/tensorflow/models/blob/5eab06018c51d0ec8bf85919bf50438e2c4056a3/slim/nets/mobilenet_v1.py#L269) to build all the variants.

File: slim/README.md (hunk 1 above)
Comment (korrawat) [inline slim/README.md]: Fixed in #1738

**Pull request merged**